homeassistant/components/notify/clicksend.py

```diff
@@ -47,10 +49,11 @@ def __init__(self, config):
         self.username = config.get(CONF_USERNAME)
         self.api_key = config.get(CONF_API_KEY)
         self.recipient = config.get(CONF_RECIPIENT)
+        self.sender = config.get(CONF_SENDER, CONF_RECIPIENT)
```

Comment:
It's better to add a schema validator that sets the default value during config validation.
```py
def validate_sender(config):
    if CONF_SENDER in config:
        return config
    config[CONF_SENDER] = config[CONF_RECIPIENT]
    return config
```
Then use this validator like this in the schema:
```py
PLATFORM_SCHEMA = vol.Schema(
    vol.All(PLATFORM_SCHEMA.extend({...}), validate_sender))
```